Choreographer_30FPS/app/src/main/jni/Common/native_app_glue/android_native_app_glue.c

```diff
@@ -0,0 +1,441 @@
+/*
+ * Copyright (C) 2010 The Android Open Source Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
```

Comment:
maybe aboid duplicate native-activity/cpufeatures files in any samples: just build a static lib like teapot/moreteapot does, and make this app's jni depends on that static lib.  currently ALL native-activity duplications have been removed; Cpufeature code are pointed to NDK's copy, no private copy are used.